packages/vuepress-theme-odyssey/styles/templates/_DocsTemplateComponent.scss

```diff
@@ -100,7 +100,12 @@
   background: var(--docs-page-bg, #{$white});
 
   @include mq(l) {
-    padding-top: $spacing-l + $spacing-m;
+    margin-top: -$spacing-m;
+    padding-top: $spacing-m;
+
+    .has-ods-beta-banner & {
+      top: ms(14);
```

Comment:
Is ms intended for positioning like this?

Reply:
Nope. @arnoldsandoval-okta I'm okay leaving this for release since it's docs-only. Can we log a ticket to use `calc()` here instead?

Reply:
It was either this or a hard value, ms felt like an OK (although not recommended) way to approach.